src/main.rs

```diff
@@ -259,39 +259,56 @@ fn uninstall(args: UninstallOpts) -> Result<(), Error> {
     );
 
     if let Some(xtensa_rust) = config.xtensa_rust {
+        config.xtensa_rust = None;
```

Comment:
All of the other installation steps have a logging statement but this one does not, was that intentional?

Reply:
It wasn't intentional, just forgot to add it! Fixed it.

Thanks for the review!